core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -42,7 +40,8 @@
  * @param <T> The concrete type of this builder.
  */
 public abstract class ManagedChannelBuilder<T extends ManagedChannelBuilder<T>> {
-  public static ManagedChannelBuilder forAddress(String name, int port) {
+  public static  ManagedChannelBuilder<? extends ManagedChannelBuilder<?>> forAddress(
```

Comment:
I was thinking it would be just `ManagedChannelBuilder<?>`. Since you do it elsewhere, it seems like you could do it here.

nit: There is a double space after `static`